src/xdocs/config.xml

```diff
@@ -869,8 +871,8 @@
 
       <source>
 &lt;!DOCTYPE module PUBLIC
-    &quot;-//Puppy Crawl//DTD Check Configuration 1.3//EN&quot;
-    &quot;http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd&quot;&gt;
+&quot;-//Puppy Crawl//DTD Check Configuration 1.3//EN&quot;
+&quot;http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd&quot;&gt;
```

Comment:
Should there be indentation +2 ?

Reply:
I did it based on the existing version (line 913)

```
      <source>
&lt;!DOCTYPE checkstyle-packages PUBLIC
&quot;-//Puppy Crawl//DTD Package Names 1.1//EN&quot;
&quot;http://checkstyle.sourceforge.net/dtds/packages_1_1.dtd&quot;&gt;
      </source>
```

Does it needs to be changed to +2 indentation after the DOCTYPE? If yes then I would do the changes at both the places?

Reply:
done